source/run-command/run-command.rst

```diff
@@ -187,7 +193,130 @@ Drivers MUST document the behavior of RunCommand if a ``maxTimeMS`` field  is al
 * See Client Side Operations Timeout's section on `runCommand <https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/client-side-operations-timeout.rst#runcommand>`_
 * See Client Side Operations Timeout's section on `runCommand behavior <https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/client-side-operations-timeout.rst#runcommand-behavior>`_
 
+
+--------------------
+``runCursorCommand``
+--------------------
+
+The following represents how a runCursorCommand API may be exposed.
```

Comment:
Not sure about the wording here. I'd reword this to clarify that `runCursorCommand` is optional, but if implemented must be compatible with the documented API. With that in mind, I'd suggest:
```suggestion
Drivers MAY expose a runCursorCommand API with the following syntax.
```

Of course, if we're requiring drivers to implement `runCursorCommand`, it should read "Drivers MUST expose [...]"